test/e2e/memcached_test.go

```diff
@@ -33,6 +33,7 @@ import (
 
 const (
 	filemode = int(0664)
+	dirmode  = 755
```

Comment:
This is the source of the permission failure you see in the tests. `755` is a base-10 integer representation, which is different than `0755`, which is an octal representation (and the one you want). If you add a leading 0 to make it octal, I think that will fix the permission issue. Or you could do what the line above does and cast an octal as an int, but I'm not sure what the value is in that. I think `filemode` would get the same value with or without the cast.

https://play.golang.org/p/OnmYc9Jf2No

Reply:
I figured Go interpreted the filemode int as `0 + int` since we use a non-prefixed int elsewhere and have no problems. I'll try this out.